Makefile

```diff
@@ -1002,16 +1011,31 @@ $(PKG): release-only
 		--root $(MACOSOUTDIR)/dist/npm \
 		--scripts ./tools/macos-installer/pkgbuild/npm/scripts \
 			$(MACOSOUTDIR)/pkgs/npm-$(NPMVERSION).pkg
+	pkgbuild --version $(NPMVERSION) \
+		--identifier org.nodejs.yarn.pkg \
+		--root $(MACOSOUTDIR)/dist/yarn \
+		--scripts ./tools/macos-installer/pkgbuild/yarn/scripts \
+			$(MACOSOUTDIR)/pkgs/yarn-$(YARNVERSION).pkg
 	productbuild --distribution $(MACOSOUTDIR)/installer/productbuild/distribution.xml \
 		--resources $(MACOSOUTDIR)/installer/productbuild/Resources \
 		--package-path $(MACOSOUTDIR)/pkgs ./$(PKG)
-	SIGN="$(PRODUCTSIGN_CERT)" PKG="$(PKG)" sh tools/osx-productsign.sh
-	sh tools/osx-notarize.sh $(FULLVERSION)
+	#SIGN="$(PRODUCTSIGN_CERT)" PKG="$(PKG)" sh tools/osx-productsign.sh
+	#sh tools/osx-notarize.sh $(FULLVERSION)
 
 .PHONY: pkg
 # Builds the macOS installer for releases.
 pkg: $(PKG)
 
+yarn-update:
+	rm -rf deps/yarn
+	mkdir -p deps/yarn
+	cd deps/yarn && wget -q https://yarnpkg.com/latest.tar.gz
+	cd deps/yarn && tar xf latest.tar.gz --strip-components=1
+	rm deps/yarn/latest.tar.gz
+	# Those shellscripts are originally meant for Yarn Windows packages; tweaked for Node Windows paths
+	perl -pi -e 's#"\$$basedir/yarn.js"#"\$$basedir/node_modules/yarn/bin/yarn.js"#g' deps/yarn/bin/yarn
+	perl -pi -e 's#"%~dp0\\yarn.js"#"%~dp0\\node_modules\\yarn\\bin\\yarn.js"#g' deps/yarn/bin/yarn.cmd
```

Comment:
Non blocking comment: there are no perl script in our Makefile, would you consider using something else instead?

Reply:
Sure, I'll use sed - I tend to use `perl` over `sed` since I found the later sometimes has compat issues on the supported flags, but if you already use it it should be fine 👍